client/client-app/EventComponent.js

```diff
@@ -7,17 +7,14 @@ import styles from "./styles";
 const EventComponent = ({name, date, time, imgSrc, description}) => {
     
     const [imgHeight, setImgHeight] = useState(0);
-    const [displayDescription, setDisplayDescription] = useState(true);
-    const [timeMarginBottom, setTimeMarginBottom] = useState(30);
+    const [descriptionText, setDescriptionText] = useState("");
```

Comment:
nit: are we able to make this optional instead of setting an empty string vs a non empty string?